src/app/pages/clients/[id].vue

```diff
@@ -71,6 +71,13 @@
               :label="$t('general.persistentKeepalive')"
             />
           </FormGroup>
+          <FormGroup>
+            <FormHeading>{{ $t('form.hooks') }}</FormHeading>
+            <FormTextField id="PreUp" v-model="data.preUp" label="PreUp" />
+            <FormTextField id="PostUp" v-model="data.postUp" label="PostUp" />
+            <FormTextField id="PreDown" v-model="data.preDown" label="PreDown" />
+            <FormTextField id="PostDown" v-model="data.postDown" label="PostDown" />
+          </FormGroup>
```

Comment:
we should add a description, that this is for the client config and that it won't work on e.g. android